src/math/j0.rs

```diff
@@ -109,6 +109,7 @@ const S02: f64 = 1.16926784663337450260e-04; /* 0x3F1EA6D2, 0xDD57DBF4 */
 const S03: f64 = 5.13546550207318111446e-07; /* 0x3EA13B54, 0xCE84D5A9 */
 const S04: f64 = 1.16614003333790000205e-09; /* 0x3E1408BC, 0xF4745D8F */
 
+/// Zeroth order of the Bessel function of the first kind (f64).
```

Comment:
I think bessel functions probably aren't common knowledge, could you add a link to https://en.wikipedia.org/wiki/Bessel_function?

Reply:
Sure!

Reply:
Done! Is this good?